.github/workflows/wheels.yml

```diff
@@ -176,7 +168,7 @@ jobs:
           # This is to prevent the build from running out of memory when
           # resolving packages on 32-bits systems (like armhf, armv7).
 
-          split -l $(expr $(expr $(cat requirements_all.txt | wc -l) + 1) / 3) requirements_all.txt requirements_all.txt
+          split -l $(expr $(expr $(cat requirements_wheels.txt | wc -l) + 1) / 3) requirements_wheels.txt requirements_wheels.txt
```

Comment:
We need to change the way we split the files as we use `-r requirements_all.txt`